source/gsoc.md

```diff
@@ -133,18 +131,17 @@ more info about mentoring, please read our [guide to being a mentor](guide-to-be
 .
 
 Prospective mentors must join the mentors@community.apache.org mailing list,
-this is where mentor specific issues are dealt with, and where
-announcements will be made. If you want to track the program
-administration you should subscribe to dev@community.apache.org.
+this is where mentor specific-issues are dealt with, and where
```

Comment:
Just curious: why it is `specific-issues` and not `specific issues`?

Reply:
I would think it's mentor-specific.